src/lib.rs

```diff
@@ -1,3 +1,95 @@
+//! A reasonably simple entity component system (ECS),
+//! with a design based on [hecs](https://github.com/Ralith/hecs).
+//!
+//! Compared to hecs, but it drops all optimisations which require additional API surface.
```

Comment:
Please compare with the updated README. I think this is a bit too much and we should focus on the difference in approach, i.e. the simpler API.

Reply:
Done.